node/src/server-modules/GlideJson.ts

```diff
@@ -381,23 +386,132 @@ export class GlideJson {
      * ```typescript
      * console.log(await GlideJson.set(client, "doc", ".", "{a: [1, 2, 3], b: {a: [1, 2], c: {a: 42}}}"));
      * // Output: 'OK' - Indicates successful setting of the value at path '.' in the key stored at `doc`.
-     * const result = await GlideJson.resp(client, "doc", "$..a");
+     * const result = await GlideJson.resp(client, "doc", {path: "$..a"});
      * console.log(result);
      * // Output: [ ["[", 1L, 2L, 3L], ["[", 1L, 2L], [42L]];
-     * console.log(await GlideJson.type(client, "doc", "..a")); // Output: ["[", 1L, 2L, 3L]
+     * console.log(await GlideJson.type(client, "doc", {path: "..a"})); // Output: ["[", 1L, 2L, 3L]
      * ```
      */
     static async resp(
         client: BaseClient,
         key: GlideString,
         options?: { path: GlideString },
-    ): Promise<ReturnTypeJson<GlideString>> {
+    ): Promise<
+        UniversalReturnTypeJson<
+            (number | GlideString) | (number | GlideString | null) | null
+        >
+    > {
         const args = ["JSON.RESP", key];
 
         if (options) {
             args.push(options.path);
         }
 
-        return _executeCommand<ReturnTypeJson<GlideString>>(client, args);
+        return _executeCommand<
+            UniversalReturnTypeJson<
+                (number | GlideString) | (number | GlideString | null) | null
+            >
+        >(client, args);
+    }
+
+    /**
+     * Returns the length of the JSON string value stored at the specified `path` within
+     * the JSON document stored at `key`.
+     *
+     * @param client - The client to execute the command.
+     * @param key - The key of the JSON document.
+     * @param options - (Optional) Additional parameters:
+     * - (Optional) path - The path within the JSON document, Defaults to root if not provided.
```

Comment:
```suggestion
     * - (Optional) `path`: The path within the JSON document, Defaults to root (`"."`) if not provided.
```
root `.` and `$` cause different results